haas/cli.py

```diff
@@ -666,43 +584,40 @@ def switch_register(switch, subtype, *args):
 @cmd
 def switch_delete(switch):
     """Delete a <switch> """
-    url = object_url('switch', switch)
-    do_delete(url)
+    check_clientlib_response(lambda: C.switch.delete(switch))
 
 
 @cmd
 def list_switches():
     """List all switches"""
-    url = object_url('switches')
-    do_get(url)
+    q = check_clientlib_response(lambda: C.switch.list())
+    sys.stdout.write('%s switches :    ' % len(q) + " ".join(q) + '\n')
 
 
 @cmd
 def port_register(switch, port):
     """Register a <port> with <switch> """
-    url = object_url('switch', switch, 'port', port)
-    do_put(url)
+    check_clientlib_response(lambda: C.port.register(switch, port))
 
 
 @cmd
 def port_delete(switch, port):
     """Delete a <port> from a <switch>"""
-    url = object_url('switch', switch, 'port', port)
-    do_delete(url)
+    check_clientlib_response(lambda: C.port.delete(switch, port))
 
 
 @cmd
 def port_connect_nic(switch, port, node, nic):
     """Connect a <port> on a <switch> to a <nic> on a <node>"""
-    url = object_url('switch', switch, 'port', port, 'connect_nic')
-    do_post(url, data={'node': node, 'nic': nic})
+    check_clientlib_response(
+            lambda: C.port.connect_nic(switch, port, node, nic)
+            )
```

Comment:
@shwsun as we discussed, I am keeping the code on single line as long as it is within 79 characters. If it exceeds it break it like above at the brackets. I test all files for any pep8 violations. This style of breaking the line of code is acceptable by those standards.